### PR TITLE
this is fix for the ticket number 21 - report inexpensive products

### LINKED ISSUE #21: REPORT: Inexpensive products
As a business aanalyst, I would like to see an HTML report showing all products priced at $999 or less

### PR DESCRIPTION
**Pull request by nrg51098**
Description of PR that completes issue here...

## Changes

- This issue is fixed by adding inexpensiveproducts.py file to handle the functionality and inexpensive.html page to show it the browser

## Requests / Responses

If this PR contains code that defines a new request/response, or changes an existing one, please put the JSON representations here.

**Request**

GET `/products/inexpensive` Creates a new product

**Response**

HTTP/1.1 200 OK

image

## Related Issues

- Fixes #21

### CODE REVIEW

File: bangazonreports/views/products/inexpensiveproducts.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            db_cursor = conn.cursor()
+
+            db_cursor.execute("""
+                select
+                    p.id,
+                    p.name,
+                    p.description,
+                    p.price
+                from bangazonapi_product p
+                where p.price <= 999
```
Comment (hannahhall) [inline bangazonreports/views/products/inexpensiveproducts.py]: What if the price is 999.50?

**Review by hannahhall — CHANGES_REQUESTED** [review]

File: bangazonreports/views/products/inexpensiveproducts.py (hunk 1 above)
Comment (nrg51098) [reply bangazonreports/views/products/inexpensiveproducts.py]: I had it <1000 but I am going based of your ticket description as 

"As a business aanalyst, I would like to see an HTML report showing all products priced at $999 or less"

But I can change if you want. Thank you

Comment (nrg51098) [reply bangazonreports/views/products/inexpensiveproducts.py]: Modified the logic to include price < 1000

**Review by hannahhall — APPROVED** [review]
👍 approved

**Pull request merged**